README.md

```diff
@@ -334,29 +334,37 @@ but you can also specify a `"fake"` agent to use a hard coded call path of searc
 
 ### Adding Documents Manually
 
-If you prefer fine grained control, and you wish to add objects to the docs object yourself (rather than using the search tool), then the previously existing `Docs` object interface can be used:
+Normally via agent execution, the agent invokes the search tool,
+which adds documents to the `Docs` object for you behind the scenes.
+However, if you prefer fine-grained control,
+you can directly interact with the `Docs` object.
+
+Note that manually adding and querying `Docs` does not impact performance.
+It just removes the automation associated with an agent picking the documents to add.
 
 ```python
+from pathlib import Path
 from paperqa import Docs, Settings
 
 # valid extensions include .pdf, .txt, and .html
-doc_paths = ("myfile.pdf", "myotherfile.pdf")
+doc_paths = (Path("myfile.pdf"), Path("myotherfile.pdf"))
```

Comment:
Can we just use plain strings? Like keeping README syntax as simple as possible

Reply:
The issue is `Docs.aadd` accepts a `Path` only. I was trying to avoid a `TypeError` in our sample code

The right long-term move is supporting `str | os.PathLike`